test/db-migrations/utils.js

```diff
@@ -0,0 +1,122 @@
+module.exports = {
+  assertIndexExists,
+  assertTableDoesNotExist,
+  assertTableSchema,
+  describeMigration,
+};
+
+const _ = require('lodash');
+const migrator = require('./migrator');
+
+function _describeMigration(describeFn, migrationName, fn) {
+  assert.strictEqual(arguments.length, 3, 'Incorrect argument count.');
+
+  assert.strictEqual(typeof describeFn, 'function');
+
+  assert.ok(migrator.exists(migrationName), `Migration '${migrationName}' already exists.`);
```

Comment:
> `Migration '${migrationName}' already exists.`

Doesn't `migrator.exists()` throw when the migration doesn't exist? Maybe I'm misunderstanding.

Reply:
Yes :+1: